students/carlos_novoa/lesson03/slicing.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+"""
+Lesson3, Slicing Lab Exercise
+"""
+
+
+def exchange_first_last(seq):
+    """first and last items exchanged"""
+    return seq[-1:] + seq[1:-1] + seq[:1]
+
+
+def remove_every_other(seq):
+    """every other item removed"""
+    return seq[::2]
+
+
+def remove_ends4_every_other(seq):
+    """first 4 and last 4 items removed, and every other in between"""
+    return seq[4:-4:2]
+
+
+def reversed(seq):
+    """elements reversed (just with slicing)"""
+    return seq[::-1]
+
+
+def thirds(seq):
+    """order: middle third, last third, first third """
+    return seq[int(len(seq) / 3):int((len(seq) / 3) * 2)] +\
+        seq[-int(len(seq) / 3):] +\
+        seq[:int(len(seq) / 3)]
```

Comment:
to make this more readable extract `int(len(seq) / 3)` into var, not everything has to be a one liner 😉